ml/jml/probabilizer.h

```diff
@@ -332,6 +336,31 @@ operator << (DB::Store_Writer & store, const GLZ_Probabilizer & prob);
 DB::Store_Reader &
 operator >> (DB::Store_Reader & store, GLZ_Probabilizer & prob);
 
+/*****************************************************************************/
+/* PROBABILIZER                                                          */
+/*****************************************************************************/
```

Comment:
Its my attempt to factor out the ML part of the probabilizer procedure. I found the whole thing a bit confused.